component/Text-Utilities/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/component/Text-Utilities/styles.css" rel="stylesheet">
+    <title>Text Utilities</title>
+</head>
+
+<body>
+
+    <!-- NAVIGATION -->
+    <nav class="nav-container">
+        <div class="nav-child">
+            <a class="nav-content"><i class="nav-logo fas fa-spa"></i></a>
+            <h2 class="nav-content nav-logo-txt">Roots</h2>
+        </div>
+        <div class="nav-child">
+            <a href="/index.html" class="nav-link nav-content">Home</a>
+            <a class="nav-link nav-content">Installation</a>
+            <a href="/component/Avatars/index.html" class="nav-link nav-content">Documentation</a>
+        </div>
+    </nav>
+    <div class="nav-body scroll">
+        <div class="nav-scroll">
+            <h4 class="nav-heading">Get Started</h4>
+            <ul class="nav-ul">
+                <li class="nav-li"><a class="nav-a">Introduction</a></li>
+                <li class="nav-li"><a class="nav-a">Colors</a></li>
+                <li class="nav-li"><a class="nav-a">Typography</a></li>
+                <li class="nav-li"><a class="nav-a">Theming</a></li>
+            </ul>
+            <h4 class="nav-heading">Components</h4>
+            <ul class="nav-ul">
+                <li class="nav-li"> <a href="/component/Avatars/index.html" class="nav-a ">Avatar</a></li>
+                <li class="nav-li"> <a href="/component/Alerts/index.html" class="nav-a ">Alerts</a></li>
+                <li class="nav-li"> <a href="/component/Badges/index.html" class="nav-a ">Badges</a></li>
+                <li class="nav-li"> <a href="/component/Buttons/index.html" class="nav-a ">Button</a></li>
+                <li class="nav-li"> <a href="/component/Cards/index.html" class="nav-a ">Card</a></li>
+                <li class="nav-li"> <a href="/component/Images/index.html" class="nav-a ">Image</a></li>
+                <li class="nav-li"> <a href="/component/Input/index.html" class="nav-a ">Input</a></li>
+                <li class="nav-li"> <a href="/component/Text-Utilities/index.html" class="nav-a ">Text Utilities</a></li>
+                <li class="nav-li"> <a href="/component/Lists/index.html" class="nav-a ">Lists</a></li>
+            </ul>
+        </div>
+    </div>
+
+    <!-- TEXT UTILITIES BODY -->
+
+
+    <div class="component-body">
+        <h2 class="component-heading">Text Utilities</h2>
+        <h5 class="component-content"></h5>
+
+
+            <h4 class="component-subheading">> Text </h4>
```

Comment:
Instead of using ">" you can use bullets

Reply:
will change it,